Readme instructions to refresh code excerpts (#6687)
flutter/website: README.md

   Your personal version of the Flutter website is now deployed to Firebase. 
   Copy the serving URL from the command output.
   
## Refreshing code excerpts

> If the build is failing with the following error: `=> ERROR: some code excerpts were not refreshed!` this means that some code excerpt(s) in the markdown files don't match up with the code in its correlated dart file. 

Reviewer: ```suggestion

A build that fails with the error `=> ERROR: some code excerpts were not refreshed!`
means that one or more code excerpts in the markdown file aren't identical to the
code in the corresponding `.dart` file. 
```

> To resolve this error, from the root of the `website` directory, run `./tool/refresh-code-excerpts.sh`. 

Reviewer: ```suggestion
To resolve this error, from the root of the `website` directory,
run `./tool/refresh-code-excerpts.sh`. 
```
